### PR TITLE
Changes to Executive Board elections

### PR DESCRIPTION
**Pull request by bmbowdish**
I made the changes to a future amendment I'll be bringing up at the house meeting on the 24th. It will be voted on at the meeting on the 31st. 

This is an amendment which will change the way we vote for Executive Positions. 

More information on the actual changes can be found here: 
https://www.youtube.com/watch?v=3Y3jE3B8HsE

### CODE REVIEW

File: articles.tex [hunk 1]
```diff
@@ -435,6 +435,14 @@
...
 
 \asection{Types of Voting}
+\asubsection{Alternative Vote}
+\asubsubsection{Method of Vote}
+Votes are cast on paper ballots, which provide a means to indicate every possible option in the vote. A ballot is then distributed to each House member eligible to cast a vote and collected in the designated ballot box for a pre-specified length of time. At the end of the voting period, the person chairing the vote collects the ballots, closing the voting period. The Vote Counters then tally the results.
+
+Voters rank the options by placing a '1' by their first choice, a '2' by their second choice, and so on, until they no longer wish to express any further preferences or run out of options.
+
+\asubsubsection{Voting Period} 
+For constitutional modification, candidate selection, and officer removal votes, the voting period must be at least forty-eight (48) hours in length. For any other type of vote, the voting period must be at least twenty-four (24) hours. The minimum length of the voting period may be explicitly lengthed, but never shortened, in the text describing the actual vote.
```
Comment (gambogi) [inline articles.tex]: > ... in the text describing the actual vote.
Is this clause meant to restrict changes to the voting period to specific to the descriptions of various votes in the constitution, or on the ballots themselves?

**Comment by liam-middlebrook** [comment]
This passed as announced in the eboard meeting on 2.4.2016 and should be merged once conflicts are resolved.
> New Voting Method for Elections PASS 41 Yes, 10 No

**Comment by gambogi** [comment]
@bmbowdish are you going to resolve these conflicts?

**Comment by bmbowdish** [comment]
Would you like these to be rebased into one commit before I merge

**Comment by gambogi** [comment]
Sure

**Comment by gambogi** [comment]
Why was this closed?

**Comment by bmbowdish** [comment]
I decided to just redo the entire thing. I don't know how any of this works and restarting was simpler.

**Pull request closed without merging**